docs/core/compatibility/6.0.md

```diff
@@ -182,4 +182,5 @@ For information on other breaking changes for containers in .NET 6, see [.NET 6
 
 ## See also
 
+- [Breaking changes in C# 10](https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%206.md)
```

Comment:
Is this intentionally an absolute URL?

Reply:
Yes. I hadn't imported the older set of breaking change docs from roslyn.

Once this one is merged, I'll watch and see if we get more traffic.

Right now, from [this doc](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes) only one gets more than 80 clicks per month. Most are in the 20s.